### DISCUSSION TITLE
Firefox renders webgl at 50%

### DISCUSSION THREAD

**Discussion by SkippyHub** [sourcelair/xterm.js#4498] [Q&A]
I believe firefox renders the xterm text as half the size compared to chrome which loads correctly.
firefox
<img width="599" alt="Screenshot 2023-04-30 at 08 45 22" src="https://user-images.githubusercontent.com/79400098/235341864-4c6e3b03-7c04-4fda-9e34-0a3ec745f4d0.png">
chrome
<img width="598" alt="Screenshot 2023-04-30 at 08 47 11" src="https://user-images.githubusercontent.com/79400098/235341967-0a4ac454-0c2e-4684-bfb3-7b91ceed3b65.png">
anyone else have this issue?

using:
  Terminal = require('xterm').Terminal;
  FitAddon = require('xterm-addon-fit').FitAddon;
  WebLinksAddon = require('xterm-addon-web-links').WebLinksAddon;
  WebglAddon = require('xterm-addon-webgl').WebglAddon;
  CanvasAddon = require('xterm-addon-canvas').CanvasAddon;

**Comment by jerch** [comment]
Cannot repro with xterm.js demo. Which xterm.js version and OS are you on? Is that from the demo? Plz also check, if your browser has some zoom setting active or not.
